scripts/index.js

```diff
@@ -0,0 +1,117 @@
+const state = {
+    temp: 82,
+    sky: "Sunny",
+    city: "Emerald City"
+};
+
+const skys = [`☁️ ☁️ ☁️ ☀️ ☁️ ☁️`,
+    `☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️`,
+    `🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧`,
+    `🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨`
+]
+
+const landscapes = [
+    `🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂`,
+    `🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷`,
+    `🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃`,
+    `🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲`
+]
```

Comment:
I like seeing these as data rather than mixed in with imperative, but we can take this even a little further by packing additional data with these strings. For instance, for the sky values, we could store pairs of the select drop-down labels with the sky strings and use that to generate the options in the select itself. Then when the selected value changes, just find the record matching the drop down value, and use that as the sky string. Completely data-driven!

The same concept could be applied to the the ground (and the desired temp color) by storing the lower bound of the temperature that would use each value, and then each time the temperature, walk through the array and find the record that the temp bins into. No hard-coded temperature logic!